key.core/src/main/antlr4/KeYParser.g4

```diff
@@ -811,3 +812,23 @@ proofScript
 
 // PROOF
 proof: PROOF EOF;
+
+// Config
+cfile: cvalue* EOF;
+//csection: LBRACKET IDENT RBRACKET;
+ckv: doc=DOC_COMMENT? ckey ':' cvalue;
+ckey: IDENT | STRING_LITERAL;
+cvalue:
+    IDENT #csymbol
+  | STRING_LITERAL #cstring
+  | BIN_LITERAL #cintb
+  | HEX_LITERAL #cinth
+  | MINUS? INT_LITERAL #cintd
+  | FLOAT_LITERAL #cfpf
+  | DOUBLE_LITERAL #cfpd
+  | REAL_LITERAL #cfpr
+  | (TRUE|FALSE) #cbool
+  | LBRACE
+     (ckv (COMMA ckv)*)? COMMA?
```

Comment:
Optional DOC_COMMENT like above?

Reply:
But `ckv` is referred to which has the DOC_COMMENT.

Currently DOC_COMMENT is unused.